src/applications/vaos/appointment-list/index.jsx

```diff
@@ -8,20 +8,21 @@ import AppointmentsPage from './components/AppointmentsPage/index';
 import RequestedAppointmentDetailsPage from './components/RequestedAppointmentDetailsPage';
 import ConfirmedAppointmentDetailsPage from './components/ConfirmedAppointmentDetailsPage';
 import CommunityCareAppointmentDetailsPage from './components/CommunityCareAppointmentDetailsPage';
+import ExpressCareDetailsPage from './components/ExpressCareDetailsPage';
+import useManualScrollRestoration from '../hooks/useManualScrollRestoration';
 
 function AppointmentListSection({ featureHomepageRefresh }) {
+  useManualScrollRestoration();
```

Comment:
This helps some of the flashing, but I need to look more into why things sometimes look jarring when moving between pages